main.py

```diff
@@ -246,7 +252,7 @@ def escape_path(path):
 		# Build qemu launch cmd
 		cmd = [xqemu_path,
 		       '-cpu','pentium3',
-		       '-machine','xbox%(accel_arg)s,bootrom=%(mcpx_path_arg)s%(short_anim_arg)s' % locals(),
+		       '-machine', 'xbox%(accel_arg)s,bootrom=%(mcpx_path_arg)s%(short_anim_arg)s,eeprom=%(eeprom_path_arg)s' % locals(),
```

Comment:
This should probably be before `%(short_anim_arg)s` so we can group the ROM image related stuff together.

Reply:
Fixed